docs/modpacks/play.md

```diff
@@ -63,3 +63,21 @@ Click the "Add a new Minecraft Core or Mod Pack" tile in the Start Menu, then cl
 ![Add a new Minecraft Core or Mod Pack Tile](../../static/img/tutorial/BakaXLAddCoreTile.png)
 
 Click "Continue" and drag the modpack file into BakaXL.
+
+### Superpack
+
+Superpack is an application for extracting/downloading modpacks to directories.
+
+#### Install through Superpack
```

Comment:
This should be separated into two sections; one for the installation method of searching for it in-app and one for manual installation.